packages/models/organizations.coffee

```diff
@@ -8,7 +8,33 @@ Organization = Astro.Class
     description: 'string'
     createdById: 'string'
 
+  events:
+    aftersave: ->
+      if @createdById
+        @addMember(@createdById)
+        profile = UserProfiles.findOne({userId: @createdById})
+        @addAdmin(profile._id)
+
   methods:
+    addMember: (userId) ->
+      UserProfiles.update({userId: userId}, {
+        $addToSet: {memberOfOrgs: @_id}
+      })
+    addAdmin: (profileId) ->
+      UserProfiles.update({_id: profileId}, {
+        $addToSet: {adminOfOrgs: @_id}
+      })
```

Comment:
Right now, we use member status to determine whether or not users can see the list of members, so an admin who is not a member wouldn't be able to see the members table or edit other members' admin status.  We should probably only allow someone to be made admin if they are a member of the organization.